data/human/deep missions.txt

```diff
@@ -2431,3 +2431,330 @@ event "battle in arneb end"
+					goto decline
+			label decline
+			`	Helm nods, knowingly, "Fair enough, star-traveler. I will handle it from here."`
+				decline
+			label accept
+			`	Helm smiles when you agree to accompany him. "This is right," he says.`
+			`	After making a few preparations, he leads you out to the harbor, where you climb aboard a small ocean-going craft that you assume belongs to him. He works on the controls for a bit and then sits down across from you as the ship veers out into the open ocean.`
+			`	"It's an old custom," he says, nodding toward the stone in your hand. "Goes back to the founding of The Deep, the first settlers. On a person's 16th birthday they are given a stone to carry with them through their lives. On special, or momentous occasions, they are to carve a few runes into it, a few cherished memories chiseled into hard stone."`
+			`	Helm pauses as a flock of jabbering sea-birds passes near the ship, indicating a nearby island, and then continues. "And then when that person dies, that stone is returned by the closest family member to the Family-Cairn."`
+			`	Helm looks up and his eyes scan the horizon. "We're almost there, gather your things." Helm stands and returns to the ship's controls.`
+			choice
+				`	"But why do I have to be the one to bring the stone to his son?"`
+					goto why
+				`	"Where do the stones come from?"`
+					goto from
+			label why
+			`	Helm shrugs as he maneuvers the ship into dock. "It is the old way. To make certain the stone is not lost, mishandled, or acquired by enemies. If the family is not near when its owner passes, it is given to a trusted friend who becomes its bearer. That task is too important and too sacred to be given to a stranger or courier. For you to take it all the way to his son, so that he may return it to earth, is right."`
+					goto earth
+			label from
+			`	Helm shrugs as he maneuvers the ship into dock. "Wherever makes the most sense at the time a stone is given. If I recall correctly Skaldgar's stone was given to him by his mother, my grandmother, from an old mine Skaldgar had been exploring on the mainland. Even at 16 he loved the earth."`
+			label earth
+			`	Helm ties the ship up and you follow him toward a grey and dour building not far from the makeshift dock. You realize this entire algae island, as small as it is, must belong to a single individual or family.`
+			`	When you arrive at the hardy and sea-stained wooden door, Helm knocks thunderously upon it and calls out, "Thorleif, open up!"`
+			`	After a brief wait the door opens ponderously, and there stands a large, red-bearded, middle aged man who looks like a younger and much healthier Skaldgar. A wide and open grin forms on his face when he sees Helm, and he gives the man a warm embrace, "Ah cousin! It's been too long, what brings you to my humble home?"`
+			`	Thorlief then notices you and frowns quizzically. "And who have you brought with you?"`
+			choice
+				`	(Answer for yourself.)`
+					goto yourself
```

Comment:
```suggestion
```
This will just lead to what's directly after the `choice` without labels needed.